src/communication/xcConnection.ts

```
@@ -18,15 +19,29 @@ export class DefaultConnection implements Connection {
         this.apis = {};
     }
 
+    getModel(xcApiName: string, serverUrl: string, getModelListener: (model: any) => void) {
+        let session = SessionFactory(serverUrl, null, null);
+        let openListener = (_: Event) => {
+            session.privateSubscriber.getModel(xcApiName, (model: any) => {
+                getModelListener(model);
+                session.close();
+            });
+        };
+        let errorListener = (_: Error) => {
```

why ignoring the error object?